tests/test_local_client.py

```
@@ -0,0 +1,322 @@
+    lives on the HTTPServer instance via server.state.
+    """
+
+    def log_message(self, format, *args):  # silence default logging
+        pass
+
+    def do_POST(self):  # noqa: N802 (BaseHTTPRequestHandler naming)
+        length = int(self.headers.get("Content-Length", "0"))
+        body = self.rfile.read(length)
+        try:
+            req = json.loads(body)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)